src/Sarif.Viewer.VisualStudio/RealKeyboard.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Windows;
+using System.Windows.Input;
+
+namespace Microsoft.Sarif.Viewer
+{
+    internal class RealKeyboard : IKeyboard
```

Comment:
I call this class `RealKeyboard` rather than just `Keyboard` because there's already a `Keyboard` class in the `System.Windows.Input` namespace, and having another class with the same name might cause confusion. OTOH, there is no existing "`IKeyboard`" interface, so that name is available for use without confusion.